### DISCUSSION TITLE
Face recognition for Sherlock

### DISCUSSION THREAD

**Discussion by henning-kln** [sherlock-project/sherlock#976] [Ideas]
English: 
My idea would be, if a profile has a profile picture, cut out the face (if there is one). Then the program can compare the face with the other possible accounts and this provides a higher accuracy of the specified accounts. So the profiles are checked whether they have the same person in the profile picture.

Sorry for my bad english skills. The most was Google Translate, but i want to publish this idea here. For the people, who speak german, this is a bit easier to understand.

German:
Meine Idee wäre es, wenn ein profil ein Profilbild hat, daraus das Gesicht (wenn es eins gibt) ausschneiden. Dann kann das Program das Gesicht mit den anderen möglichen Accounts vergleichen und so eine Höhere Genauigkeit der angegebenen Accounts liefern. So werden die Profile überprüft ob sie die gleiche Person im Profilbild hat.

**Comment by AayushBoliya** [comment]
Hoye mate! I'm already working on a similar project, Idk Sherlock would implement it, but I want to create a Face Browser thing where you can use a photograph to search a person on internet using Face Recognition tech. Would you like to help?

**Reply by nurudeen196** [reply]
Yeah if I can

**Reply by henning-kln** [reply]
>Would you like to help?

Yes man. If you need help contact me over Telegram.
https://t.me/henningkln

**Reply by BHughes1** [reply]
Hi there, you might want to look into this project I read an article about a while ago. I'm not sure if it is still active, but this idea is something I can get behind if you still need help. The project is called Social Mapper. Here is the github  https://github.com/Greenwolf/social_mapper

Let me know what you think and if you need help building something like this I'd be willing to participate.

If you have WickrMe you can talk to me there. My username is fletcher80

**Reply by AayushBoliya** [reply]
Thanks for replying, No matter how much you explore GitHub, there's always lot of gold that you've missed!
I'll try to use it now.

**Reply by Kataphractoi** [reply]
The next step is to use AI to put a face to a name such that one person can use an image and it will search names or use names and it will search images or even just as a name to face converter or a face to name converter. joking ofc that'll take too much effort to be feasible. Could make an AI boosted version as a premium beta testing product idk.
